explorer/src/utils/explorer.ts

```diff
@@ -139,8 +148,12 @@ const nativeExplorerContractUri = (
     } else if (chainId === chainIDs["aurora"]) {
       base = "https://aurorascan.dev/address/";
     } else if (chainId === chainIDs["alephium"]) {
-      // FIXME
-      base = "https://explorer.testnet.alephium.org/address"
+      nativeAddress = addressFromContractId(nativeAddress)
```

Comment:
I put this code out of `getNativeAddress` because we have two formats for the contract, `ContractId`(hex string) and `ContractAddress`(base58). We want to use different formats for different cases, and in `getNativeAddress` we can only handle one case.